docs/src/pages/docs/Connecting to Databases/trino.mdx

```diff
@@ -8,20 +8,80 @@ version: 1
 
 ## Trino
 
-Supported trino version 352 and higher
-
-The [sqlalchemy-trino](https://pypi.org/project/sqlalchemy-trino/) library is the recommended way to connect to Trino through SQLAlchemy.
-
-The expected connection string is formatted as follows:
+SuperSet supports Trino >=352 via SQLAlchemy by using the [sqlalchemy-trino](https://pypi.org/project/sqlalchemy-trino/) library.
```

Comment:
The correct spelling is "Superset", not "SuperSet"
```suggestion
Superset supports Trino >=352 via SQLAlchemy by using the [sqlalchemy-trino](https://pypi.org/project/sqlalchemy-trino/) library.
```

Reply:
:) I just though `SuperSet` is right style

Reply:
Resolved